app/views/posts/_comment_card.html.erb

```diff
@@ -1,7 +1,8 @@
 <div class="comment-container">
 
     <% if comments.any? %>
-        <% comments.each do |comment| %>
+        <% comments.includes(:user).each do |comment| %>
+
```

Comment:
- [ ] Kindly notice that fixing the N+1 issue in the views doesn't actually fix it due to the fact that data had already been fetched from the database. To fix the problem, you'll need to implement the solution inside of the controller which then takes effect when the data is fetched which fixes the problem. Kindly use the `includes` method inside of the Posts controller and not in the views.